openrtx/include/rtx/OpMode_M17.hpp

```diff
@@ -148,6 +149,10 @@ class OpMode_M17 : public OpMode
     M17::M17Demodulator  demodulator;  ///< M17 demodulator.
     M17::M17FrameDecoder decoder;      ///< M17 frame decoder
     M17::M17FrameEncoder encoder;      ///< M17 frame encoder
+    M17::M17LinkSetupFrame lsf;        ///< M17 link setup frame
+
+
+    static constexpr int16_t MINIMUM_GPS_TIME_TICKS = 5 * 30; // Wait at least 5 seconds between GPS transmissions
```

Comment:
Do you mind moving this above all the variables' declarations? Usually I tend to have:
- private member functions
- private constants
- private variables